### DISCUSSION TITLE
Modeling complex combustion involving multiple regimes in FDS

### DISCUSSION THREAD

**Discussion by vijay00901** [firemodels/fds#13431] [Q&A]
Dear friends, 

I am trying to simulate backdraft phenomena using LES and finite rate chemistry in FDS. backdraft involves complex combustion processes (pre-mixed and non-pre-mixed). Fire is predominantly non-premixed. As such, if an initial unmixed fraction of 0 (fully pre-mixed) is used, equation 5.18 in the technical reference guide becomes zero. That is, the state inside a cell is always completely mixed. 

i.e.  equation 5.19 becomes 
![comb](https://github.com/user-attachments/assets/a158a770-42aa-4b5b-bc37-2af94741c8b7)

Isn't this assumption wrong given a fire is not pre-mixed? 

If an initial unmixed fraction of 1 (default) is chosen, then the time variation of the unmixed fraction is governed by eq 5.18. Isn't this approach more appropriate? 

Also, when using finite rate chemistry and an initial unmixed fraction of 1, is this model equivalent to EDC or extended EDC? Thank you so much.

**Comment by rmcdermo** [comment]
Your question is confusing me.  You first state, correctly, that backdraft phenomena involve at least partially premixed flames.  The initial unmixed fraction is a tunable parameter for the initial state of mixing in the cell during the combustion substep (obviously, this is a simplification and reality involves transported evolution of the state of mixing [a mixing progress variable]).

> Isn't this assumption wrong given a fire is not pre-mixed?

You are the one defining the state of combustion.  Yes, usually we think of a fire plume as a diffusion flame, but your application is backdraft.  The conflagration may initiate as a premixed flame kernel.  There is currently no option in FDS to make the initial unmixed fraction vary in space or time.

> If an initial unmixed fraction of 1 (default) is chosen, then the time variation of the unmixed fraction is governed by eq 5.18. Isn't this approach more appropriate?

More appropriate for what?  If you have a diffusion flame, then yes.  Do you have a pure diffusion flame?

> Also, when using finite rate chemistry and an initial unmixed fraction of 1, is this model equivalent to EDC or extended EDC?

Extended EDC.  With infinitely fast chemistry the model is what I call basic EDC or what other like to call EDM.

**Reply by vijay00901** [reply]
Dear Dr. McDermott, 

Assuming I am simulating completely premixed combustion and using finite rate chemistry by defining Arrhenius parameters,  shouldn't I give the initial unmixed fraction as 0? Does choosing this parameter as 0.5 make the combustion partially premixed?

Another question I have is why this initial unmixed fraction is made a tunable parameter in FDS? Please clarify.

**Reply by rmcdermo** [reply]
If the flow were perfectly laminar, then yes.  But these flows are turbulent, and so products curl around and the flame sheet starts to mix the premixed region with products.  So, tuning of zeta_0 is not unreasonable.  The unmixed fraction is a mixing progress variable.  Strictly speaking we should solve a transport equation for it, but we take the simple approach because it reduces to the usual EDM model or Burke-Schumann relations.  For now, that is what we have.  You are welcome to rethink the approach.

**Comment by vijay00901** [comment]
Dear Dr. McDermott, 

As per the technical reference guide, the turbulent batch reactor is equivalent to extended EDC when the unmixed fraction is held constant.  Just to confirm, in my model, unmixed fraction varies with time, but the chemistry is finite rate. This is also treated as extended EDC right? Thank you so much.

**Reply by rmcdermo** [reply]
Well, all this naming is semantics.  "Extended EDC" to means you are doing both turbulent mixing and finite-rate chemistry.  Whether you treat the reactor like a perfectly stirred reactor (PSR), which is a steady state reactor, or a batch reactor (time dependent), like we do in FDS, is your call.
